proteobench/io/params/i2masschroq.py

```diff
@@ -58,8 +66,10 @@ def extract_params(fname: pathlib.Path) -> ProteoBenchParameters:
         allowed_miscleavages=max_cleavage,
         min_peptide_length=None,  # "spectrum, minimum fragment mz"
         max_peptide_length=None,  # Not mentioned, up to 38 AA in peptides
-        fixed_mods=",".join(params.loc[params.index.str.contains("residue, modification mass")].dropna()),
-        variable_mods=",".join(params.loc[params.index.str.contains("residue, potential modification mass")].dropna()),
+        fixed_mods=",".join(params.loc[params.index.str.contains(
```

Comment:
so it's really an extra.

Reply:
yep. These modifications are added in the search if the "quick" mod parameter is on.